packages/flutter/lib/src/cupertino/context_menu.dart

```diff
@@ -37,6 +45,13 @@ typedef ContextMenuPreviewBuilder = Widget Function(
   Widget child,
 );
 
+/// A function that builds the child and handles the transition between the
+/// default child and the preview when the CupertinoContextMenu is open.
+typedef ContextMenuBuilder = Widget Function(
```

Comment:
Nit: Maybe name this CupertinoContextMenuBuilder, since "context menu" is also the name of the right click menus.